### DISCUSSION TITLE
Is Autodiff symbolic?

### DISCUSSION THREAD

**Discussion by bharswami** [autodiff/autodiff#340] [General]
Are derivatives computed using forward/backward/central difference schemes? Or are the derivatives computed symbolically using the function handle given to the "derivative" function?
Thanks

**Comment by allanleal** [comment]
autodiff does not use finite differences to compute the derivatives. It's also not a symbolical derivative calculator. Please check the website (https://autodiff.github.io) and tutorials and this will become more evident.
